src/librustc_lint/lib.rs

```diff
@@ -207,6 +207,10 @@ pub fn register_builtins(store: &mut lint::LintStore, sess: Option<&Session>) {
             id: LintId::of(LIFETIME_UNDERSCORE),
             reference: "RFC 1177 <https://github.com/rust-lang/rfcs/pull/1177>",
         },
+        FutureIncompatibleInfo {
+            id: LintId::of(SAFE_EXTERN_STATICS),
+            reference: "issue 35112 <https://github.com/rust-lang/rust/issues/35112>",
```

Comment:
the new policy is to make a separate tracking issue, following roughly [this template]; if you could update with an issue like that, it'd be great.

[this template]: https://github.com/rust-lang/rfcs/blob/master/text/1589-rustc-bug-fix-procedure.md#tracking-issue-template